src/Entity/User.php

```diff
@@ -189,4 +195,24 @@ public function setHomepage(?string $homepage): self
 
         return $this;
     }
+
+    /**
+     * @return string|null
+     */
+    public function getLinkedIn(): ?string
+    {
+        return $this->linkedIn;
+    }
+
+    /**
+     * @param string|null $linkedIn
+     *
+     * @return User
+     */
+    public function setLinkedIn(?string $linkedIn): User
```

Comment:
```suggestion
    public function setLinkedIn(?string $linkedIn): self
```
Yra gera praktika naudoti `self` kaip `type hint`ą, o ne konkrečią klasę.
Tada tvarkant kodą (skaldant, sujungiant, perkeliant) yra daug lengviau.
Pvz.
```php
class PublicRelationsUser extends User
{ 
//    public function setLinkedIn(?string $linkedIn): User
}

$prUser = (new PublicRelationsUser())->setLinkedIn('https://www.linkedin.com/in/aurelijusbanelis');
if ($prUser instanceof PublicRelationsUser) 
   // Čia vis dar bus PublicRelationsUser, nors dokumentacija sakytų, kad tai yra tėvinė klasė.
}
```